src/plugins/plugin.legend.js

```diff
@@ -95,6 +95,18 @@ function getBoxWidth(labelOpts, fontSize) {
 		labelOpts.boxWidth;
 }
 
+/**
+ * Helper function to get the box height
+ * @param {object} labelOpts - the label options on the legend
+ * @param {*} fontSize - the label font size
+ * @return {number} height of the color box area
+ */
+function getBoxHeight(labelOpts, fontSize) {
+	return labelOpts.usePointStyle || isNullOrUndef(labelOpts.boxHeight) ?
```

Comment:
Should we be consistent with width/height? I'd actually change `getBoxWidth` to have similar logic with this new one, the `> fontSize` is unexpected.

Reply:
I wasn't sure about that. `getBoxWidth` doesn't explicitly handle the case of `boxWidth` set to `null` or `undefined` (I guess that does remove the box eventually by setting the width to null/undef). I can easily update this though, I don't think anything depends on it